components/patient-data/api/src/main/java/org/phenotips/data/Feature.java

```diff
@@ -91,4 +92,19 @@
      * @todo move to VocabularyProperty and/or implement a VocabularyProperty-to-PhenotipsPropertyName mapping service
      */
     String getValue();
+
+    /**
+     * Returns a feature property name. Constructed from type property prefixing the {@code negative_} prefix for
```

Comment:
I would also document WHAT is property name (e.g. is it "property name as stored in PhenoTips"?) property is a very generic term